src/classes/CON_Email_TEST.cls

```diff
@@ -55,6 +55,24 @@ private class CON_Email_TEST {
         insert listCon;
 	}
 
+	@isTest static void testSingleEmailSmartSet() {
+
+		Contact newCont = new Contact(
+			FirstName = 'JohnnyTest',
+			LastName = 'JohnnyTest',
+			WorkEmail__c = 'workit@test.com'
+		);
+
+		Test.startTest();
+        insert newCont;
```

Comment:
Since we cannot directly duplicate the scenario of converting a lead this is the next best thing. Since we are inserting a new Contact with only one email field having a value this is the same process that a lead goes through when being converted.